pkgs/top-level/all-packages.nix

```diff
@@ -15974,6 +15974,9 @@ with pkgs;
   cargo-edit = callPackage ../development/tools/rust/cargo-edit {
     inherit (darwin.apple_sdk.frameworks) Security;
   };
+  cargo-leptos = callPackage ../development/tools/rust/cargo-leptos {
+    inherit (darwin.apple_sdk.frameworks) Security;
```

Comment:
You will need to include CoreServices here, or just take `darwin` as the package argument, and then inherit the relevant frameworks in a `let ... in ...` binding.